src/calendar.scss

```diff
@@ -0,0 +1,57 @@
+@import "./variables/variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  width: $calendar-width;
+  margin: auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.